### PR TITLE
Installed cucumber, typescript, playwright and added simple test

### PR DESCRIPTION
**Pull request by YahyoMahmudov**

### CODE REVIEW

File: src/test/steps/loginSteps.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    page = await browser.newPage();
+    await page.goto("https://px-stage.self-service.danads.com");
+});
+
+Given('User clicks login button', async function () {
+    //await page.locator("//a/*[text()='Log in']").click();
+})
+
+Given('User enters username', async function () {
+    page.locator("(//input[@name='username'])[2]").type("Username");
```
Comment (jkalandarov) [inline src/test/steps/loginSteps.ts:18]: Locators should be stored in page objects, aren't they?

File: src/test/features/login.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Feature: Login
+
+Background: 
+ Given User navigates to the application
+ And User clicks login button
+
+ Scenario: Login to Core
+        Given User enters username
```
Comment (jkalandarov) [inline src/test/features/login.feature:8]: which user? Admin, publisher, advertiser, or agency?

File: src/test/steps/loginSteps.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+});
+
+When('User enters log in with email button', async function () {
+    //page.locator("//span[text()=' Log in with email ']").click();
+});
+
+When('User enters password', async function () {
+    page.locator("(//input[@name='password'])[2]").type("Password");
```
Comment (jkalandarov) [inline src/test/steps/loginSteps.ts:26]: It's not a good idea to expose secrets (passwords). Also, the password in Cognito is different.

File: src/test/steps/loginSteps.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+});
+
+When('User clicks Login button', async function () {
+    page.locator("(//input[@type='Submit'])[2]").click();
+});
+
+Then('User is in Dashboard page', async function () {
+    console.log("User logged in")
```
Comment (jkalandarov) [inline src/test/steps/loginSteps.ts:34]: console.log doesn't perform any action other than printing message to the terminal. Step name says `User is in Dashboard page`, so you need to verify that the user has been navigated to Dashboard page.

**Review by jkalandarov — CHANGES_REQUESTED** [review]

File: src/test/steps/loginSteps.ts (hunk 1 above)
Comment (YahyoMahmudov) [reply src/test/steps/loginSteps.ts:18]: Yes, they should be. But this is just a sample test script.

**Review by jkalandarov — APPROVED** [review]

**Pull request merged**